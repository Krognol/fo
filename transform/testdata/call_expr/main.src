package main

type A[T] T

func main() {
	var a0 A[string] = "foo"
	print(a0)
	var a1 A[int] = 42
	print(a1)
	a2 := A[bool](true)
	func(b bool) {}(a2)
}
