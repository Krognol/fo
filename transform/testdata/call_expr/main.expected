package main

type A interface{}

func main() {
	var a0 A = "foo"
	print(a0.(string))
	var a1 A = 42
	print(a1.(int))
	a2 := A(true)
	func(b bool) {}(a2.(bool))
}
